### ISSUE TITLE
Example doesn't work!

### ISSUE THREAD

**Issue by vipullal-github** [flutter/website#11555]
### Page URL

https://docs.flutter.dev/cookbook/networking/web-sockets/

### Page source

https://github.com/flutter/website/tree/main/src/content/cookbook/networking/web-sockets.md

### Describe the problem

Problem: The example given does not tally with the example given on GitHub. For starters, the code uses wss protocol while the example of the server given at https://github.com/dart-lang/shelf/blob/master/pkgs/shelf_web_socket/example/example.dart does not use Secure connection.

I corrected this and now I get an error on the client side ("Connection not upgraded). I believe this means that the server did not send the upgrade response. I need to dig through the code to see why. 

Basically, I am trying to implement a WebSocket server and client. I am using the shelf and the shelf_web_socket plugin on the server. The client code is from the problem page.  

It would be wonderful if the code examples given work out of the box.

### I would like to fix this problem.

- [X] I will try and fix this problem on docs.flutter.dev.

**Comment by vipullal-github** [comment]
I think I made a mistake in my assessment
